gateway-service-impl/src/main/java/org/hypertrace/gateway/service/explore/TheRestGroupRequestHandler.java

```diff
@@ -168,39 +171,39 @@ private ExploreRequest createRequest(
       ExploreRequest originalRequest, ExploreResponse.Builder originalResponse) {
     Filter.Builder filterBuilder = Filter.newBuilder();
     filterBuilder.setOperator(Operator.AND);
-    List<String> groupByColumns = groupByColumnList(originalRequest);
+    Map<String, Expression> groupBySelectionExpressionsByKey =
+        groupByExpressionByKey(originalRequest);
```

Comment:
should we rename `groupByExpressionByKey` -> `groupByExpressionByResultKey` and `groupBySelectionExpressionsByKey` -> `groupBySelectionExpressionsByResultKey`?

Reply:
Sure, will use result name for consistency.

Reply:
renamed.